tests/app/phase-team-response/issues-responded/issues-responded.component.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { IssuesRespondedComponent } from '../../../../src/app/phase-team-response/issues-responded/issues-responded.component';
+import { Issue, STATUS } from '../../../../src/app/core/models/issue.model';
+import { ISSUE_WITH_EMPTY_DESCRIPTION } from '../../../constants/githubissue.constants';
+import { IssueService } from '../../../../src/app/core/services/issue.service';
+import { UserService } from '../../../../src/app/core/services/user.service';
+import { USER_Q, TEAM_4 } from '../../../constants/data.constants';
+
+describe('IssuesRespondedComponent', () => {
+  describe('.ngOnInit()', () => {
+    const DUMMY_TEAM = TEAM_4;
+    const DUMMY_DUPLICATE_ISSUE_ID = 1;
+    const DUMMY_RESPONSE = 'dummy response';
+    let dummyIssue: Issue;
+
+    const issueService = new IssueService(null, null, null, null, null, null);
+    const userService = new UserService(null, null);
+    userService.currentUser = USER_Q;
+    const issuesRespondedComponent = new IssuesRespondedComponent(issueService, userService);
+    issuesRespondedComponent.ngOnInit();
+
+    beforeEach(() => {
+        dummyIssue = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_EMPTY_DESCRIPTION, DUMMY_TEAM);
+        issueService.updateLocalStore(dummyIssue);
+    });
+
+    it('should set filter to return false for a non-duplicate issue with no team response and not marked done', () => {
```

Comment:
I think we should mark the issue as done for this test to test for "no team response" boolean properly!
(So that we know it is the "no team response" part that is causing problems and not the "not marked done" part)

Reply:
My idea here is that I would like to structure the tests in this way

| hasTeamResponse | !isDuplicate | isDone|
| --- | --- | --- |
| F | * | * |
| T | F | * |
| T | T | F |
| T | T | T |

Since the short-circuiting behaviour means that `isDone` does not really affect overall result of the evaluation.

What do you think?

Reply:
Sorry for the late reply! I missed your comment. I think what you said makes sense, it seems alright to go ahead with it!